graphics/VectorRenderer.h

```diff
@@ -375,6 +378,16 @@ class VectorRenderer {
 			_gradientFactor = factor;
 	}
 
+	/**
+	 * Sets the pixel interval for drawing shadows
+	 * 
+	 * @param shadowIntensity interval for drawing shadows
+	 */
+	virtual void setShadowIntensity(uint32 shadowIntensity) {
+		if (shadowIntensity > 0)
+			_shadowIntensity = shadowIntensity;
```

Comment:
I would write `else warning("setShadowIntensity(): zero intensity")`, so the devs could catch a badly written code